source/analytics/index.html.erb

```diff
@@ -119,27 +104,43 @@ show_features: true
         </div>
         <div class="col-md-5">
           <h3>Agent Utilization Chart with Easy Drill-Down</h3>
-
           <p>
             Agent utilization chart shows you agents with the highest utilization. You can drill down to the jobs with the highest wait time
             for a chosen agent and further down to the individual job instances of a chosen job to see their build and wait times.
           </p>
         </div>
       </div>
+    </div>
 
-      <div class="hr"></div>
-
+    <div class="testimonial-section">
+      <div class="testimonial-photo"></div>
+      <div class="testimonial-text"> <p>"Your valuable support enhanced our experience with the GoCD application. I cannot say enough how much of a pleasure it has been working with each of your team members while troubleshooting production issues in our environment, without causing any discontinuities or issues to end users." <br /> <span class="bold">Vijay Mohan, Sr. Devops Engineer, Barclaycard - USA</span></p>
+      </div>
     </div>
 
-    <div class="cta-section">
-      <div class="cta-row">
-        <% link_to '/enterprise/contact/index.html' do %>
-          <div class="cta-button">
-            <h3>GET STARTED NOW</h3>
-          </div>
-        <% end %>
+    <div class="form-section contact-form_analytics" >
+      <div class="container">
+        <div class="centered-content-container">
+          <h1 class="form-header"> Start Your Free 30-Day Trial Support </h1>
+          <script src="//app-e.marketo.com/js/forms2/js/forms2.min.js"></script>
+          <form id="mktoForm_9095"></form>
+          <div class="form-loading-error"></div>
+          <script defer="true" src="//app-e.marketo.com/js/forms2/js/forms2.min.js"></script>
+          <script defer="true" src="/assets/javascripts/marketo_popover.js" onload= "new MarketoForm('//app-e.marketo.com', '199-QDE-291').init(9095)"></script>
+          <noscript>
+            <div class="error-msg-container">
+              <p class="error-msg_text">
+                Our signup form is blocked by your browser. You can email us directly at support@thoughtworks.com.
+              </p>
+            </div>
+            <style>
```

Comment:
Why is there an inline style here?

Reply:
Noscript tag doesnt take style written in Stylesheet. It needs Inline style

Reply:
@arvindsv  The style for footer I have written should only take effect when the JS is turned off thats that reason it is inline style  inside noscript.

Reply:
That doesn't explain the code I pasted above. But, it's fine.